retux/client/mixins.py

```diff
@@ -188,3 +188,35 @@ async def get(cls, bot: "Bot", path: str, **query_params: dict | None) -> dict:
         """
         route = _Route(method=_RouteMethod.GET, path=path)
         return await bot.http.request(route, payload=query_params)
+
+
+class Serializable:
+    """
+    A mixin for structures that need to be generated from a `dict`.
+
+    This class exists to simplify compatibility with attrs converters.
+
+    Methods
+    -------
+    convert(json: dict) : `object`
+        Converts a `dict provided by Discord to an instance
+        of the class. Used for compatibility with attrs 
+        converters.
+    """
+    @classmethod
+    async def convert(cls, json: dict) -> object:
```

Comment:
```suggestion
    def _c(cls, json: dict) -> object:
```

What's asynchronous exactly?